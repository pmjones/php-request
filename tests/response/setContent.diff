001+ bool(false)
001- bool(true)