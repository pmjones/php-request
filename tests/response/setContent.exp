bool(true)
string(3) "foo"