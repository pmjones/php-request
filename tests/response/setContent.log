
---- EXPECTED OUTPUT
bool(true)
string(3) "foo"
---- ACTUAL OUTPUT
bool(false)
string(3) "foo"
---- FAILED
