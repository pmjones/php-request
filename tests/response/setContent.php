<?php
$response = new SapiResponse();
var_dump($response->setContent('foo') === $response);
var_dump($response->getContent());
