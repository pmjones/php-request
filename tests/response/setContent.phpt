--TEST--
SapiResponse::setContent
--FILE--
<?php
$response = new SapiResponse();
var_dump($response->setContent('foo') === $response);
var_dump($response->getContent());
--EXPECT--
bool(true)
string(3) "foo"
