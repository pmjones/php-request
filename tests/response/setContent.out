bool(false)
string(3) "foo"